### PR TITLE
README: update sponsors

### PR DESCRIPTION
**Pull request by selsta**
- http -> https
- remove outdated sponsors
- add new sponsors
- add link to sponsorships page

### CODE REVIEW

**Review by dEBRUYNE-1 — APPROVED** [review]

**Review by erciccione — COMMENTED** [review]
Isn't better to link to the sponsors page on getmonero? There are more details there and we wouldn't need to maintain two lists.

**Comment by selsta** [comment]
Linking to sponsors will make them kinda invisible, this way the logo shows at least.

File: README.md [hunk 1]
```diff
@@ -42,13 +42,11 @@ The Bitcoin donation address is: `1KTexdemPdxSBcG55heUuTjDRYqbC5ZL8H`
...
-[<img width="80" src="https://static.getmonero.org/images/sponsors/mymonero.png"/>](https://mymonero.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/kitware.png?1"/>](http://kitware.com)
-[<img width="100" src="https://static.getmonero.org/images/sponsors/dome9.png"/>](http://dome9.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/araxis.png"/>](http://araxis.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/jetbrains.png"/>](http://www.jetbrains.com/)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/navicat.png"/>](http://www.navicat.com/)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/symas.png"/>](http://www.symas.com/)
+[<img width="150" src="https://www.getmonero.org/img/sponsors/tarilabs.png"/>](https://tarilabs.com/)
+[<img width="150" src="https://www.getmonero.org/img/sponsors/globee.png"/>](https://globee.com/)
+[<img width="150" src="https://static.getmonero.org/images/sponsors/symas.png"/>](https://www.symas.com/)
```
Comment (erciccione) [inline README.md:47]:
```suggestion
[<img width="150" src="https://www.getmonero.org/img/sponsors/symas.png"/>](https://www.symas.com/)
```

**Review by erciccione — COMMENTED** [review]
In that case i would still add a link to the Sponsorships page at the bottom of this section. Saying that more details can be found there

**Comment by selsta** [comment]
Added a link.

File: README.md [hunk 2]
```diff
@@ -40,15 +40,13 @@ The Monero donation address is: `888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnj
...
-GUI development funding and/or some supporting services are also graciously provided by sponsors:
-
-[<img width="80" src="https://static.getmonero.org/images/sponsors/mymonero.png"/>](https://mymonero.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/kitware.png?1"/>](http://kitware.com)
-[<img width="100" src="https://static.getmonero.org/images/sponsors/dome9.png"/>](http://dome9.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/araxis.png"/>](http://araxis.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/jetbrains.png"/>](http://www.jetbrains.com/)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/navicat.png"/>](http://www.navicat.com/)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/symas.png"/>](http://www.symas.com/)
+GUI development funding and/or some supporting services are also graciously provided by [sponsors](https://web.getmonero.org/community/sponsorships/):
```
Comment (erciccione) [inline README.md:43]:
```suggestion
GUI development funding and/or some supporting services are also graciously provided by [sponsors](https://www.getmonero.org/community/sponsorships/):
```

Same for the PR in `monero`

Comment (selsta) [reply README.md]: done

**Review by erciccione — APPROVED** [review]

**Pull request merged**